spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
```

Comment:
@backus I concur with @dkubb and would do this across the board as a separate PR. 

@dkubb is this mergeable now?